config/generic_domains.php

```diff
@@ -33,3 +33,826 @@
+    '.ae',
+    '.ae',
+    '.aero',
+    '.af',
+    '.af',
+    '.ag',
+    '.agency',
+    '.ai',
+    '.airforce',
+    '.al',
+    '.allfinanz',
+    '.alsace',
+    '.am',
+    '.am',
+    '.android',
+    '.ao',
+    '.apartments',
+    '.app',
+    '.aquarelle',
+    '.ar',
+    '.archi',
+    '.army',
+    '.arpa',
+    '.as',
+    '.asia',
+    '.associates',
+    '.at',
+    '.attorney',
+    '.au',
+    '.au',
+    '.auction',
+    '.audio',
+    '.autos',
+    '.aw',
+    '.ax',
+    '.baby',
+    '.band',
+    '.bank',
+    '.bar',
+    '.barclaycard',
+    '.barclays',
+    '.bargains',
+    '.bayern',
+    '.be',
+    '.beer',
+    '.berlin',
+    '.best',
+    '.bg',
+    '.bh',
+    '.bi',
+    '.bi',
+    '.bid',
+    '.bike',
+    '.bingo',
+    '.bio',
+    '.biz',
+    '.biz',
+    '.biz',
+    '.bj',
+    '.bj',
+    '.black',
+    '.blackfriday',
+    '.blog',
+    '.blue',
+    '.bm',
+    '.bmw',
+    '.bn',
+    '.bnpparibas',
+    '.bo',
+    '.boo',
+    '.boutique',
+    '.br',
+    '.br',
+    '.brussels',
+    '.bt',
+    '.budapest',
+    '.build',
+    '.build',
+    '.builders',
+    '.business',
+    '.buzz',
+    '.bw',
+    '.by',
+    '.bz',
+    '.bz',
+    '.bzh',
+    '.ca',
+    '.cab',
+    '.cal',
+    '.cam',
+    '.camera',
+    '.camp',
+    '.cancerresearch',
+    '.canon',
+    '.capetown',
+    '.capital',
+    '.cards',
+    '.care',
+    '.career',
+    '.careers',
+    '.casa',
+    '.cash',
+    '.casino',
+    '.cat',
+    '.catering',
+    '.cc',
+    '.cc',
+    '.center',
+    '.ceo',
+    '.cern',
+    '.cf',
+    '.ch',
+    '.channel',
+    '.chat',
+    '.cheap',
+    '.christmas',
+    '.chrome',
+    '.church',
+    '.ci',
+    '.city',
+    '.cl',
+    '.claims',
+    '.cleaning',
+    '.click',
+    '.clinic',
+    '.clothing',
+    '.cloud',
+    '.club',
+    '.cm',
+    '.cn',
+    '.cn',
+    '.co',
+    '.uk.co',
+    '.coach',
+    '.codes',
+    '.coffee',
+    '.college',
+    '.college',
+    '.cologne',
+    '.com',
+    '.*.com',
+    '.*.com',
+    '.community',
+    '.company',
+    '.computer',
+    '.condos',
+    '.construction',
+    '.consulting',
+    '.contractors',
+    '.cooking',
+    '.cool',
+    '.coop',
+    '.country',
+    '.courses',
+    '.cr',
+    '.credit',
+    '.creditcard',
+    '.cricket',
+    '.cruises',
+    '.cuisinella',
+    '.cx',
+    '.cymru',
+    '.cz',
+    '.dad',
+    '.dance',
+    '.dating',
+    '.datsun',
+    '.day',
+    '.dclk',
+    '.de',
+    '.deals',
+    '.degree',
+    '.delivery',
+    '.democrat',
+    '.dental',
+    '.dentist',
+    '.desi',
+    '.design',
+    '.dev',
+    '.dev',
+    '.diamonds',
+    '.diet',
+    '.digital',
+    '.direct',
+    '.directory',
+    '.discount',
+    '.dk',
+    '.dm',
+    '.dm',
+    '.do',
+    '.docs',
+    '.domains',
+    '.doosan',
+    '.durban',
+    '.dvag',
+    '.dz',
+    '.earth',
+    '.eat',
+    '.ec',
+    '.*.ec',
+    '.edu',
+    '.edu',
+    '.edu',
```

Comment:
This TLD is duplicated, but it could be changed to ".edu.*" to support domains like ".edu.pa".